### ISSUE TITLE
Issue with alpha 9

### ISSUE THREAD

**Issue by ulfbearth** [expired6978/SKSE64Plugins#22]
the new skee64.dll is not loading bodymorphs for body, but it does with clothing. I'm using SAM morphs for racemenu btw...

**Comment by bliblablob** [comment]
I can confirm this issue with Alpha 9. All meshes which include the body mesh (and including the body mesh itself) do not get any morphs applied. Meshes without the body mesh seem to work fine.

**Comment by MoltenM** [comment]
Yes, getting same issue. I had CTDs as described in Issue #20, with Alpha 9 that fixed them but now sliders are only applied to clothes. Spawned multiple bandits, and they all had different body types, but when unclothed they had zeroed sliders.

**Comment by expired6978** [comment]
I cannot reproduce this locally, can you send me a package containing a save, and the appropriate mods?
